app/javascript/packages/document-capture/components/status-message.tsx

```diff
@@ -7,19 +7,13 @@ export const Status = {
   ERROR: 'ERROR',
```

Comment:
Somewhat related to my previous comment, this might be a good opportunity to use [TypeScript native enums](https://www.typescriptlang.org/docs/handbook/enums.html)? It's one of the few language "features" of TypeScript that isn't strictly typings-related.

Reply:
If you do opt for an enum, I'd suggest using `const` enum, which can have a pretty dramatic effect on the generated output size of the code ([related PR](https://github.com/18F/identity-idp/pull/9270), [documentation](https://www.typescriptlang.org/docs/handbook/enums.html#const-enums)).